File: ProjectTrackervb/PTServiceClient/ResourceName.vb
Imports System.Windows.Forms

Public Class ResourceName

  Public Sub New(ByVal id As String, ByVal name As String)

    InitializeComponent()
    Me.IdLabel1.Text = id
    Me.NameLabel1.Text = name

  End Sub

  Private Sub OK_Button_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles OK_Button.Click
    Me.DialogResult = System.Windows.Forms.DialogResult.OK
    Me.Close()
  End Sub

  Private Sub Cancel_Button_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Cancel_Button.Click
    Me.DialogResult = System.Windows.Forms.DialogResult.Cancel
    Me.Close()
  End Sub

End Class

File: Samples/CslaNet/cs/ProjectTrackercs/PTServiceClient/ResourceName.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace PTServiceClient
{
  public partial class ResourceName : Form
  {
    public ResourceName(string id, string name)
    {
      InitializeComponent();
      this.IdLabel1.Text = id;
      this.NameLabel1.Text = name;
    }

    private void OK_Button_Click(object sender, EventArgs e)
    {
      this.DialogResult = DialogResult.OK;
      this.Close();
    }

    private void Cancel_Button_Click(object sender, EventArgs e)
    {
      this.DialogResult = DialogResult.Cancel;
      this.Close();
    }
  }
}